src/api/app/models/buildresult.rb

```diff
@@ -98,4 +92,12 @@ def self.summary(project_name)
 
     local_build_results
   end
+
+  private
+
+  def code2index(code)
```

Comment:
is that code being used anywhere?

Reply:
Yep, in [this line](https://github.com/openSUSE/open-build-service/blob/25aafabde00ca52eeabde5c756a2628535da42ca/src/api/app/models/buildresult.rb#L94).

Reply:
argh, you are right my grep gave me the following output:

```
src/api/app/models/buildresult.rb
60:  def self.code2index(code)
94:      build.summary.sort! { |a, b| code2index(a.code) <=> code2index(b.code) }
```

and for me it looked like it was being used in the same method, but it's not!